components/admin_console/feature_discovery/features/index.ts

```diff
@@ -25,5 +24,4 @@ export {
     GuestAccessFeatureDiscovery,
     SystemRolesFeatureDiscovery,
     GroupsFeatureDiscovery,
-    PermissionsFeatureDiscovery,
```

Comment:
Probably out of scope for this PR but should the feature discovery now be targeted specifically for team permission schemes that is still an licensed feature?

Reply:
> Probably out of scope for this PR but should the feature discovery now be targeted specifically for team permission schemes that is still an licensed feature?

Yes, that's definitely an option. I think design may need to come up with something net new for this since it doesn't necessarily fit the current pattern of a menu item with a full feature discovery page (unless a new menu item were added for this). 

//cc @aaronrothschild would this be your domain to own?